.github/workflows/php.yml

```diff
@@ -60,5 +60,8 @@ jobs:
     - name: Run PHPUnit
       run: vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
 
+    - name: Run Infection
+      run: vendor/bin/infection run --verbose --show-mutations --only-covered --no-interaction --min-msi=100 --min-covered-msi=100
```

Comment:
`min-msi` and `min-covered-msi` are better places in `infection.json.dist` so you don't need to duplicate the args every time you run infection

Reply:
@bendavies great point, it wasn't possible to do via config when I started using Infection, serves me right to just copy-paste my stuff ever since. :) Would you be interested in finishing here? :crossed_fingers: